javascript-data-model/order-history.js

```diff
@@ -1,49 +1,60 @@
 var orderHistory = [
   {
     orderPlaced: 'August 4, 2020',
-    totalPrice: '$34.00',
+    orderTotalPrice: '$34.00',
     expectedDelivery: 'August 8, 2020',
-    itemCategory: 'book',
-    returnWindowClosed: 'September 7, 2020',
-    itemName: 'JavaScript for impatient programmers',
+    itemDescription: {
+      category: 'book',
+      itemName: 'JavaScript for impatient programmers',
+      priceBeforeTax: '$31.55'
+    },
+    orderReturnWindowClosedDate: 'September 7, 2020',
     shipTo: 'JS Masher',
-    orderNumber: '114-3941689-8772232',
-    priceBeforeTax: '$31.55'
-
+    orderNumber: '114-3941689-8772232'
   },
   {
     orderPlaced: 'July 19, 2020',
-    totalPrice: '$44.53',
+    orderTotalPrice: '$44.53',
     expectedDelivery: 'July 20, 2020',
-    itemCategory: 'book',
-    returnWindowClosed: 'August 19, 2020',
-    itemName: 'The Timeless Way of Building',
+    itemDescription: {
+      category: 'book',
+      itemName: 'The Timeless Way of Building',
+      priceBeforeTax: '$41.33'
+    },
+    orderReturnWindowClosedDate: 'August 19, 2020',
     shipTo: 'JS Masher',
-    orderNumber: '113-9984268-1280257',
-    priceBeforeTax: '$41.33'
-
+    orderNumber: '113-9984268-1280257'
   },
   {
     orderPlaced: 'July 4, 2020',
-    totalPrice: '$17.22',
+    orderTotalPrice: '$17.22',
     expectedDelivery: 'July 7, 2020',
-    itemCategory: 'electronics',
-    returnWindowClosed: 'August 5, 2020',
-    itemName: 'Gamecube Controller Adapter. Super Smash Bros Switch Gamecube Adapter for WII U, PC. Support Turbo and Vibration Features. No Driver and No Lag-Gamecube Adapter',
+    itemDescription: {
+      category: 'electronics',
+      itemName: 'Gamecube Controller Adapter. Super Smash Bros Switch Gamecube Adapter for WII U, PC. Support Turbo and Vibration Features. No Driver and No Lag-Gamecube Adapter',
+      priceBeforeTax: '$41.33'
+    },
+    orderReturnWindowClosedDate: 'August 5, 2020',
     shipTo: 'JS Masher',
-    orderNumber: '114-2875557-9059409',
-    priceBeforeTax: '$15.98'
+    orderNumber: '114-2875557-9059409'
 
   },
   {
     orderPlaced: 'July 3, 2020',
-    totalPrice: '$138.93',
+    orderTotalPrice: '$138.93',
     expectedDelivery: 'July 5, 2020',
-    itemCategory: 'book electronics',
-    returnWindowClosed: 'August 4, 2020',
-    itemName: 'GameCube Controller - Super Smash Bros. Edition (Nintendo Switch), The Art of Sql',
+    item1Description: {
+      category: 'electronics',
+      itemName: 'GameCube Controller - Super Smash Bros. Edition (Nintendo Switch)',
+      priceBeforeTax: '$94.95'
+    },
+    item2Description: {
+      category: 'book',
+      itemName: 'The Art of Sql',
+      priceBeforeTax: '$33.99'
+    },
```

Comment:
Numbered properties are not the right approach. WHat if the order has 100 items? That would be painful to organize. If you find yourself numbering properties, that is usually an indicator that you have a list of data. What data type works well to render lists?